### DISCUSSION TITLE
Download progression

### DISCUSSION THREAD

**Discussion by onxipole** [FreeTubeApp/FreeTube#4319] [Q&A] [answered]
Hi there, sorry if I am asking something already answered by someone.
Is there anyway to show download progression of a video inside FT ?

Thank you

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @onxipole ik going to link you to other discussions because this was indeed already answered in #4191 and https://github.com/FreeTubeApp/FreeTube/discussions/4269

**Comment by onxipole** [comment, marked as answer]
Sorry, I didn't find that. Well, let's wait, then, and thank you again for your tremendous work, very much appreciated ! Farewell !
